src/Core/src/Core/IFrameworkElement.cs

```diff
@@ -22,7 +22,5 @@ public interface IFrameworkElement
 
 		double Width { get; }
 		double Height { get; }
-
-		Thickness Margin { get; }
```

Comment:
We may still want the margin here as if we do something like a shape, we still want that to have a margin. What was your thought behind the move?

Reply:
Mainly that @hartez told me to move it :-)

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/13878#discussion_r584893478